lib/webrtcstreamer.js

```diff
@@ -42,15 +42,15 @@ export function filteredEvents(pendingEvents) {
   return currentClick ? [...evs, currentClick] : evs;
 }
 
-const fetchAPI = (url, data) =>
+const fetchAPI = (url, data, errorHandler = (err) => console.log(err)) =>
   fetch(url, {
     method: data ? 'POST' : 'GET',
     body: data ? JSON.stringify(data) : undefined
   }).then(resp => {
     if (resp.ok) {
       return resp.json();
     }
-    throw new Error('Network response was not ok.');
+    errorHandler('Network response was not ok.');
```

Comment:
we may also want to add a `catch` block for this promise (just realized we dont catch any errors thrown from the `fetch` which happens sometimes when the request goes bonkers. 

so: `fetch(...).then(...).catch(errorHandler)`